### DISCUSSION TITLE
Artisan: Phidget Manager Could Not be Launched

### DISCUSSION THREAD

**Discussion by nathanjsolomon** [artisan-roaster-scope/artisan#642] [Q&A]
I am running a windows 10 machine. I've JUST downloaded everything onto my computer from the Phidget Drivers and Library to the Latest Artisan software.

I am able to launch the phidget device manager manually and can see that it is reading the temperature just fine. It is updating and responsive. 

However, when I close it out and run artisan I get " 13:33:27.783 Exception: PhidgetManager couldn't be started. Verify that the Phidget driver is correctly installed!"

I know the driver is correctly installed. Not only did I just install it, but it's working fine in the Phidget manager. My only thought at this point is that it is looking for the Phidget Manager in the wrong folder and cannot find the launcher. Currently, the phidget manager is in my Downloads folder, and I do NOT have access to the C drive program files folder to put it where it is probably trying to find it.

Does anybody have any ideas? Is there a way to change where Artisan is trying to find the Phidget Launcher to the folder I'm using?

**Comment by MAKOMO** [comment]
Artisan is accessing the Phidget hardware via the Phidget driver that needs to be installed on the machine  for system wide access. Just run the Phidget installer to get the driver installed correctly. Make sure that the account running Artisan has enough privileges to access the Phidget driver and the externally connected hardware via its USB port.
